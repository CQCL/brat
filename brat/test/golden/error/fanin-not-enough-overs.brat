f(Nat, Nat) -> Vec(Nat, 3)
f = { [\/] }
