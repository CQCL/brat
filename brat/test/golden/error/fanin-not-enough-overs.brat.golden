Error in test/golden/error/fanin-not-enough-overs.brat@FC {start = Pos {line = 2, col = 5}, end = Pos {line = 2, col = 13}}:
f = { [\/] }
    ^^^^^^^^

  Type error: Not enough inputs to make a vector of size 3

